src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -1,6 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StringResourcesPath>$(MSBuildProjectDirectory)\..\src\Resources\Strings.resx</StringResourcesPath>
```

Comment:
What are these resources needed for in the test project?

Reply:
Because in #92185 we have added a new test that uses pinvokes to verify the results:

https://github.com/dotnet/runtime/blob/018efc57d557b5234e168b5c16e6ea1153de6db2/src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs#L858

and the easiest way to get it working was to include these 3 files:

https://github.com/dotnet/runtime/blob/018efc57d557b5234e168b5c16e6ea1153de6db2/src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj#L65-L70

and at least one of them uses the resource that is missing in the test project:

https://github.com/dotnet/runtime/blob/018efc57d557b5234e168b5c16e6ea1153de6db2/src/libraries/Common/src/Interop/OSX/Interop.libproc.cs#L371

I've merged a PR which introduced a copy of the resource:

https://github.com/dotnet/runtime/blob/018efc57d557b5234e168b5c16e6ea1153de6db2/src/libraries/System.Diagnostics.Process/tests/Resources/Strings.resx#L126-L128

while I believe that better solution would be to simply include the src resource in the test project.